format-avro/src/test/java/io/cdap/plugin/format/avro/input/AvroInputFormatProviderTest.java

```diff
@@ -70,4 +71,36 @@ public void testSchemaDetection() throws IOException {
     Schema schema = schemaDetector.detectSchema(avroFile.getPath(), formatContext, Collections.emptyMap());
     Assert.assertEquals(expectedSchema, schema);
   }
+
+  @Test
+  public void testPathFieldSchemaDetection() throws IOException {
```

Comment:
this should be named something like testPathFieldConflict, and there should be another test that actually tests that the schema is generated correctly when the path field is set and the data does not have the path field.

Reply:
Added another test to add pathField if set when data does not have.CHanged test name to testPathFieldConflict